src/App/Tracker/Controller/Hooks/Listeners/AbstractListener.php

```diff
@@ -325,4 +321,117 @@ protected function createStatus(Github $gitHub, TrackerProject $project, $issueN
 			$status->state, $status->targetUrl, $status->description, $status->context
 		);
 	}
+
+	/**
+	 * Set Categories
+	 *
+	 * @param   object       $hookData       Hook data payload
+	 * @param   Logger       $logger         Logger object
+	 * @param   object       $project        Object containing project data
+	 * @param   IssuesTable  $table          Table object
+	 * @param   array        $addCategories  The Categories to add
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 * @throws  \RuntimeException
+	 */
+	protected function setCategories($hookData, Logger $logger, $project, IssuesTable $table, $addCategories)
+	{
+		// The Github ID if we have a pull or issue so that method can handle both
+		$issueNumber = $this->getIssueNumber($hookData);
+
+		if ($issueNumber === null)
+		{
+			$message = sprintf('Error retrieving issue number for %s/%s', $project->gh_user, $project->gh_project);
+
+			$logger->error($message);
+
+			throw new \RuntimeException($message);
+		}
+
+		$categoryModel            = new CategoryModel($this->getContainer()->get('db'));
```

Comment:
Ahh. I guess it is just a c&p issue from the old issue :( I'm going to fix it thanks!